packages/@react-types/shared/src/events.d.ts

```diff
@@ -30,7 +30,7 @@ export type FocusEvent = BaseEvent<ReactFocusEvent<any>>;
 export type PointerType = 'mouse' | 'pen' | 'touch' | 'keyboard';
 
 export interface PressEvent {
-  type: 'pressstart' | 'pressend' | 'press',
+  type: 'pressstart' | 'pressend' | 'pressup' | 'press',
```

Comment:
what's the difference between pressup and pressend?

Reply:
press up is like mouse up: fired whenever the user releases a pointer over the target. press end is fired whenever the user's pointer leaves the target or is released. press is fired if there is a pointer down event followed by a pointer up event over the target.